readme.txt

```diff
@@ -1,9 +1,9 @@
 === Pressbooks ===
 Contributors: Pressbooks <code@pressbooks.com>
 Tags: ebooks, publishing, webbooks
-Requires at least: 5.4
-Tested up to: 5.5.2
-Requires PHP: 7.1
+Requires at least: 5.5.3

```

Comment:
I believe we can set the requires at least to 5.5 so users don't get a nag if their wordpress is not up-to-date.

Reply:
I'd prefer to set it to 5.5.3 as it's a stable release and the one we use in our production bedrocks.